org.opent2t.sample.thermostat.superpopular/org.opent2t.sample.thermostat.superpopular.json

```diff
@@ -91,6 +91,18 @@
       "$ref": "oic.baseResource.json#/definitions/oic.r.baseresource"
     },
     {
+      "$ref": "oic.r.mode.json#/definitions/oic.r.mode"
+    },
+    {
+      "$ref": "oic.r.clock.json#/definitions/oic.r.clock"
+    },
+    {
+      "$ref": "oic.r.temperature.json#/definitions/oic.r.temperature"
+    },
+    {
+      "$ref": "oic.r.humidity.json#/definitions/oic.r.humidity"
+    },
```

Comment:
That isn't my understanding. I believe this is where we actually describe all of the things that we want this schema to represent. Above we describe a 'definition' for org.opent2t.sample.thermostat.superpopular, and we reference that here, along with oic.core and oic.r.baseresource (which, looking at it, I'm not sure makes sense here either).

For example, by including oic.r.temperature here, I believe that's saying we want to return an object that looks like:

{
  "id": "<id>",
  "rt": "org.opent2t.sample.thermostat.superpopular",
  "ambientTemperature": { <temperature> }
  . . . (the rest of the Thermostat properties)
  "temperature": 52
  "units": "F"
}

Which isn't what we're trying to describe.

---
In reply to: [79681760](https://github.com/openT2T/translators/pull/102#discussion_r79681760) [](ancestors = 79681760,79676179)